components/status_dropdown/index.jsx

```diff
@@ -9,8 +9,12 @@ import {getCurrentUser, getStatusForUserId} from 'mattermost-redux/selectors/ent
 import {Preferences} from 'mattermost-redux/constants';
 import {get} from 'mattermost-redux/selectors/entities/preferences';
 
+import {getUserTimezone} from 'mattermost-redux/selectors/entities/timezone';
+
 import {openModal} from 'actions/views/modals';
 
+import {areTimezonesEnabledAndSupported} from 'selectors/general';
+
 import StatusDropdown from 'components/status_dropdown/status_dropdown.jsx';
```

Comment:
please group imports logically/contextually (all mattermost-redux imports in one block and so on)